recipes/dask-searchcv/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dask-searchcv" %}
+{% set version = "0.0.1" %}
+{% set md5 = "7d8fb38ba01d7518e2d22e5933f1e419" %}
```

Comment:
Is this required? I just went with the easier way of grabbing the md5 from pypi directly.